src/Microsoft.Diagnostics.Monitoring.RestServer/Models/EventPipeConfigurationModel.cs

```diff
@@ -16,6 +17,7 @@ public class EventPipeConfigurationModel
         public bool RequestRundown { get; set; } = true;
 
         [DataMember(Name = "bufferSizeInMB")]
+        [Range(1, int.MaxValue)]
```

Comment:
I was thinking we would not allow this to be higher than some reasonable memory limit. If the value is very large, it is likely to just shut down the container or monitoring agent app.